aerospike/metadata.csv

```diff
@@ -352,14 +371,14 @@ aerospike.namespace.total_bytes_disk,gauge,,byte,,The total bytes of disk space
 aerospike.namespace.total_bytes_memory,gauge,,byte,,The total bytes of memory allocated to this namespace on this node. Deprecated since 3.9,0,aerospike,,
 aerospike.namespace.used_bytes_disk,gauge,,byte,,The total bytes of disk space used by this namespace on this node. Deprecated since 3.9,0,aerospike,,
 aerospike.namespace.used_bytes_memory,gauge,,byte,,The total bytes of memory used by this namespace on this node. Deprecated since 3.9,0,aerospike,,
-aerospike.namespace.hwm_breached,gauge,,,,"If true, Aerospike has breached 'high-water-[disk/memory]-pct' for this namespace. Deprecated since 3.9",0,aerospike,,
-aerospike.namespace.stop_writes,gauge,,,,"If true, this namespace is currently not allowing writes.",0,aerospike,,
+aerospike.namespace.hwm_breached,gauge,,,,[OpenMetricsV2] "If true Aerospike has breached 'high-water-[disk/memory]-pct' for this namespace. Deprecated since 3.9",0,aerospike,,
```

Comment:
```suggestion
aerospike.namespace.hwm_breached,gauge,,,,"[OpenMetricsV2] If true Aerospike has breached 'high-water-[disk/memory]-pct' for this namespace. Deprecated since 3.9",0,aerospike,,
```